lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -472,13 +472,28 @@ isSubrangeOf r1 r2 =
 -- | Represent stake pool identifier. Note that the internal representation is
 -- left open currently, until we figure out a better type for those.
 newtype PoolId = PoolId
-    { getPoolId :: Text }
+    { getPoolId :: Digest Blake2b_160 }
```

Comment:
Probably wrong here as suggested by Johannes in https://github.com/input-output-hk/cardano-wallet/issues/713 .

Reply:
Are we sure about the digest size here :thinking: ? Jörmungandr has been quite fond of `blake2b256` so far. Also, since this is quite backend-specific, perhaps we should better keep the pool id type rather flexible (hence the `Text` originally).